RELEASE_NOTES.md

```diff
@@ -55,28 +55,34 @@ Produced programs are fully self-sufficient and do not need JVM or other runtime
 
 On macOS it also requires Xcode 11.0 or newer to be installed.
 
-The language and library version supported by this release match Kotlin 1.4.
+The language and library version supported by this release match Kotlin 1.5.
 However, there are certain limitations, see section [Known Limitations](#limitations).
 
  Currently _Kotlin/Native_ uses reference counting based memory management scheme with a cycle
 collection algorithm. Multiple threads could be used, but objects must be explicitly transferred
-between threads, and same object couldn't be accessed by two threads concurrently.
+between threads, and same object couldn't be accessed by two threads concurrently unless it is frozen.
+See the relevant [documentation](https://kotlinlang.org/docs/reference/native/concurrency.html).
```

Comment:
May be mention blog-post about new GC/MM?

Reply:
Done.